<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RsaKeysFrame</class>
 <widget class="QFrame" name="RsaKeysFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>RSA Keys</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="keysLabel">
        <property name="text">
         <string>RSA private keys are loaded from a file or PKCS #11 token.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="keysView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addFileButton">
          <property name="text">
           <string>Add new keyfile…</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addItemButton">
          <property name="text">
           <string>Add new token…</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteItemButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove key</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="libsLabel">
        <property name="text">
         <string>PKCS #11 provider libraries.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="libsView">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>54</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="addLibraryButton">
          <property name="text">
           <string>Add new provider…</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteLibraryButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove provider</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
